src/components/RoomNameInput/index.js

```diff
@@ -73,6 +73,7 @@ class RoomNameInput extends Component {
                 autoCapitalize="none"
                 onBlur={this.props.onBlur}
                 autoFocus={this.props.autoFocus}
+                maxLength={CONST.REPORT.MAX_ROOM_NAME_LENGTH}
```

Comment:
@puneetlath I think this should be set to `index.native.js` too, on native user can still provide a valid room name that pass 80 char and he gets same error (which is misleading error)

Reply:
That's right. I didn't add it in `index.native.js` because it was already there 😄

Reply:
my bad I didn't noticed it was there , sorry for this

Reply:
All good!